docs/configuration-reference/platforms/baremetal.md

```diff
@@ -107,6 +107,8 @@ cluster "bare-metal" {
   install_to_smallest_disk = "false"
 
   kernel_args = var.kernel_args
+
+  download_protocol = "http"
```

Comment:
We should show example for https. Don't you think?

Reply:
Since it's a cached_install, the line should be deleted.

Maybe it also makes sense to add a comment here or to the docs that using `http` can help if the iPXE https boot is too slow or does not work because of newer SSL settings